README.md

```diff
@@ -12,6 +12,15 @@
 ```yml
 gems:
   - jekyll-sitemap
+
+# optionally, to include static files
+
+sitemap:
+  extensions:
+    - .html  # this is the default
+    - .htm   # this is the default
+    - .xhtml # this is the default
+    - .pdf
```

Comment:
1. Why do we need the preceding `.`? Would there ever be an extension without one? can we normalize it and make it optional?

2. For the docs: Can you explain what this feature does and when I'd want to use it? If I want to add one additional extension, do I have to add all the defaults back in or is it cumulative? (if it's not, when would I *not* want to include the defaults?)